test/end2end_test.go

```diff
@@ -7809,24 +7809,14 @@ func (s) TestStreamingServerInterceptorGetsConnection(t *testing.T) {
 	}
 	defer ss.Stop()
 
-	respParam := []*testpb.ResponseParameters{
-		{
-			Size: int32(1),
-		},
-	}
-	payload, err := newPayload(testpb.PayloadType_COMPRESSABLE, int32(1))
-	if err != nil {
-		t.Fatal(err)
-	}
-	req := &testpb.StreamingOutputCallRequest{
-		ResponseType:       testpb.PayloadType_COMPRESSABLE,
-		ResponseParameters: respParam,
-		Payload:            payload,
-	}
 	ctx, cancel := context.WithTimeout(context.Background(), defaultTestTimeout)
 	defer cancel()
-	_, err = ss.Client.StreamingOutputCall(ctx, req)
-	if status.Code(err) != codes.OK {
-		t.Fatalf("%v.StreamingOutputCall(_) = _, %v, want _, error code %s", ss.Client, err, codes.OK)
+
+	s, err := ss.Client.StreamingOutputCall(ctx, &testpb.StreamingOutputCallRequest{})
+	if err != nil {
+		t.Fatalf("%v.StreamingOutputCall(_) = _, %v, want _, <nil>", ss.Client, err)
```

Comment:
I wouldn't log the client here: there isn't anything useful in it to understanding the test failure, and it will make it hard to see the actual error.  Just `Fatalf("ss.Client.StremaingOutputCall(_) = _, %v; want _, <nil>", err)` is good.  (Same below.)